<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stock_MainWindow</class>
 <widget class="QMainWindow" name="stock_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>721</width>
      <height>611</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>进货信息查询</string>
        </attribute>
        <widget class="QComboBox" name="comboBox">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>101</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>订单基本信息：</string>
         </property>
        </widget>
        <widget class="QTableView" name="tableView">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>90</y>
           <width>341</width>
           <height>391</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>330</x>
           <y>60</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>订单详细内容：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>20</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>订单号查询：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>20</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="geometry">
          <rect>
           <x>610</x>
           <y>20</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
        <widget class="QTableView" name="tableView_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>261</width>
           <height>391</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_10">
         <property name="geometry">
          <rect>
           <x>220</x>
           <y>520</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>取消订单</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_11">
         <property name="geometry">
          <rect>
           <x>594</x>
           <y>520</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>取消订单条目</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>进货计划制定</string>
        </attribute>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>81</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>供货商：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>30</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>商品信息：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>260</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>进货清单：</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>360</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>删除记录</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>430</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>生成订单</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>290</y>
           <width>461</width>
           <height>221</height>
          </rect>
         </property>
        </widget>
        <widget class="QTableView" name="tableView_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>221</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
        <widget class="QTableView" name="tableView_5">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>50</y>
           <width>351</width>
           <height>192</height>
          </rect>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>10</y>
           <width>71</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_12">
         <property name="geometry">
          <rect>
           <x>500</x>
           <y>10</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>剩余仓库空间：</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_9">
         <property name="geometry">
          <rect>
           <x>594</x>
           <y>250</y>
           <width>81</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>加入进货清单</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_13">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>530</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>进货总价格：</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_2">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>530</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>530</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>进货总运费：</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="lcdNumber_3">
         <property name="geometry">
          <rect>
           <x>420</x>
           <y>530</y>
           <width>131</width>
           <height>23</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>供货商管理</string>
        </attribute>
        <widget class="QComboBox" name="comboBox_2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>121</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>60</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>供货商基本信息：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>70</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>供货商商品信息：</string>
         </property>
        </widget>
        <widget class="QTableView" name="tableView_4">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>90</y>
           <width>281</width>
           <height>351</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>120</x>
           <y>60</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>60</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>供货商名字：</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_10">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>60</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>供货商地址：</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>390</x>
           <y>60</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_11">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>90</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>供货商提供的商品：</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>130</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_5">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>180</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>增加商品</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>240</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>删除商品</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>574</x>
           <y>30</y>
           <width>91</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>修改供应商信息</string>
         </property>
        </widget>
        <widget class="QTableWidget" name="tableWidget">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>120</y>
           <width>491</width>
           <height>361</height>
          </rect>
         </property>
        </widget>
        <widget class="QTableView" name="tableView_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>90</y>
           <width>231</width>
           <height>351</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_8">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>290</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>删除供应商</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
